release/sbin/deploy-all.sh

```diff
@@ -132,7 +132,7 @@ function download_spark {
       if [[ -e "${spark_tar}" ]]; then
         echo "Skip downloading openmldbspark as ${spark_tar} already exists"
       else
-        url="https://github.com/4paradigm/spark/releases/download/v3.2.1-openmldb${OPENMLDB_VERSION}/${spark_tar}"
+        url="https://github.com/4paradigm/spark/releases/download/v3.2.1-openmldb0.7.1/${spark_tar}"
```

Comment:
why use the fixed version?

Reply:
It is for testing and has been reverted.